packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerVariables.ts

```diff
@@ -1,3 +1,41 @@
-export interface DatepickerVariables {}
+import { pxToRem } from '../../../../utils';
 
-export const datepickerVariables = (siteVariables): DatepickerVariables => ({});
+export interface DatepickerVariables {
+  datepickerCalendarCellHeight: string;
```

Comment:
Please remove the `datepicker` prefix. We do not prefic the variables by the component name anywhere.

Reply:
Sorry about that. Fixed.